packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.component.js

```diff
@@ -16,7 +16,7 @@ function adaptOnSort(onChange) {
 }
 
 function ListToVirtualizedList(props) {
-	const { id, items, columns, titleProps } = props;
+	const { id, items, columns, itemProps, sort, titleProps } = props;
```

Comment:
Why not also add display mode to the spread and put it in the argument definition?